bears/stylus/StylintBear.py

```diff
@@ -0,0 +1,219 @@
+    """
+    Attempts to catch little mistakes (duplication of rules for instance) and
+    enforces a code style guide on Stylus (a dynamic stylesheet language
+    with the ``.styl`` extension that is compiled into CSS) files.
+
+    The ``StylintBear`` is able to catch following problems:
+    - Duplication of rules
+    - Mixed spaces and tabs
+    - Unnecessary brackets
+    - Missing colon between property and value
+    - Naming conventions
+    - Trailing whitespace
+    - Consistent quotation style
+    - Use of extra spaces inside parenthesis
+    - Naming convention when declaring classes, ids, and variables
+    - Unnecessary leading zeroes on decimal points
+    - Checks if a property is valid CSS or HTML
+    """
+
+    LANGUAGES = {'Stylus'}
+    REQUIREMENTS = {NpmRequirement('stylint', '1.5.9')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Formatting', 'Syntax', 'Redundancy'}
+    SEE_MORE = 'https://github.com/SimenB/stylint'
+
+    @staticmethod
+    def generate_config(filename, file,
+                        allow_block_keyword: bool = False,
+                        allow_brackets: str = 'never',
+                        allow_colons: str = 'always',
+                        allow_colors: str = 'always',
+                        allow_spaces_after_commas: str = 'always',
+                        allow_spaces_after_comments: str = 'always',
+                        allow_trailing_whitespace: str = 'never',
+                        allow_css_literal: str = 'never',
+                        max_selector_depth: bool = False,
+                        check_duplicates: bool = True,
+                        allow_efficient_properties: str = 'always',
+                        set_extend_preference: bool = False,
+                        indent_size: bool = False,
+                        allow_leading_zero: str = 'never',
+                        set_max_errors: bool = False,
+                        set_max_warnings: bool = False,
+                        mixed_spaces_and_tabs: bool = False,
+                        set_naming_convention: bool = False,
+                        set_strict_naming_convention: bool = False,
+                        allow_none_keyword: str = 'never',
+                        check_no_important_keyword: bool = True,
+                        allow_spaces_inside_parens: bool = False,
+                        allow_placeholder: str = 'always',
+                        prefix_vars_with_dollar: str = 'always',
+                        allow_semicolons: str = 'never',
+                        sort_order: str = 'alphabetical',
+                        allow_stacked_properties: str = 'never',
+                        check_valid_property: bool = True,
+                        allow_zero_units: str = 'never',
+                        z_index_normalize: bool = False,
+                        stylint_config: str = ''):
```

Comment:
`styling_config: str=''` (no space around `=`)
This is PEP8, so we should follow.
Same applies for all other params above (sorry :3)